examples/aci.yml

```
@@ -207,7 +207,7 @@ common_queries:
         help_text: The current Pulse-Width Modulation value of the fan.
     labels:
       - property_name: dn
-        regex: "^topology/pod-(?P<pod>[1-9][0-9]*)/node-(?P<node>[1-9][0-9]*)/sys/ch/(?P<slotType>[a-z]+)-(?P<slot>[^/]+)/"
+        regex: "^topology/pod-(?P<pod>[1-9][0-9]*)/node-(?P<node>[1-9][0-9]*)/sys/ch/(?P<slotType>[a-z]+)-(?P<slot>[^/]+)/ft/fan-(?P<fanId>[1-9][0-9]*)/"
```

For consistency with other label names, where I've avoided an `ID` suffix, I'd simplify `fanId` to just `fan`.
```suggestion
        regex: "^topology/pod-(?P<pod>[1-9][0-9]*)/node-(?P<node>[1-9][0-9]*)/sys/ch/(?P<slotType>[a-z]+)-(?P<slot>[^/]+)/ft/fan-(?P<fan>[1-9][0-9]*)/"
```

---

Good point, it's corrected !